tests/typ/visualize/gradient-aspect-ratio.typ

```diff
@@ -0,0 +1,22 @@
+// Tests that the aspect ratio is taken into account to correct
+// the angle of the gradient.
+
+---
+#set page(height: 120pt, width: 110pt, margin: 0pt)
+#set block(spacing: 0pt)
+
+#for i in range(0, 360, step: 30) {
```

Comment:
could you apply 2 space indent everywhere and add newlines to the end of all files?